starboard/android/shared/starboard_bridge.cc

```diff
@@ -109,7 +109,8 @@ extern "C" SB_EXPORT_PLATFORM void JNI_StarboardBridge_HandleDeepLink(
 
   auto* manager = cobalt::browser::DeepLinkManager::GetInstance();
   if (applicationReady) {
```

Comment:
This isn't part of this CL, but I think `applicationReady` is a bit inaccurate, shouldn't it be `applicationRunning` or `applicationStarted` ?